code/modules/mob/living/carbon/human/species/machine.dm

```diff
@@ -127,9 +127,21 @@
 			if((head_organ.dna.species.name in tmp_hair.species_allowed) && (robohead.company in tmp_hair.models_allowed)) //Populate the list of available monitor styles only with styles that the monitor-head is allowed to use.
 				hair += i
 
+		var/file = file2text("config/custom_sprites.txt")		//Pulls up the custom_sprites list
+		var/lines = splittext(file, "\n")
+
+		for(var/line in lines)									// Looks for lines set up as screen:ckey:screen_name
+			var/list/Entry = splittext(line, ":")				// split lines
+			for(var/i = 1 to Entry.len)
+				Entry[i] = trim(Entry[i])						// Cleans up lines
+				if(Entry.len != 3 || Entry[1] != "screen")		// Ignore entries that aren't for screens
+					continue
+				if(Entry[2] == H.ckey)							// They're in the list? Custom sprite time, var and icon change required
+					hair += Entry[3]							// Adds custom screen to list
+
 		var/new_style = input(H, "Select a monitor display", "Monitor Display", head_organ.h_style) as null|anything in hair
 		if(H.incapacitated())
 			to_chat(src, "<span class='warning'>You were interrupted while changing your monitor display.</span>")
 			return
 		if(new_style)
-			H.change_hair(new_style)
\ No newline at end of file
+			H.change_hair(new_style, 1)							// The 1 is to enable custom sprites
```

Comment:
If you are wondering why I added a 1 here, it is so it can use fluff items in addition to normal screen styles. I have tested this, and it does not effect normal screens nor does it allow for people to use fluff hair if they don't it set in custom_sprites